spring-boot/proxy/src/test/java/eu/coatrack/proxy/security/local_api_key_manager_tests/AbstractLocalApiKeyManagerSetup.java

```diff
@@ -0,0 +1,50 @@
+package eu.coatrack.proxy.security.local_api_key_manager_tests;
+
+/*-
+ * #%L
+ * coatrack-proxy
+ * %%
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import eu.coatrack.api.ApiKey;
+import eu.coatrack.proxy.security.ApiKeyFetcher;
+import eu.coatrack.proxy.security.LocalApiKeyManager;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+
+public class AbstractLocalApiKeyManagerSetup {
```

Comment:
Maybe make this class `abstract`?
Maybe change its name to indicate that it is a base class for test classes, e.g. `LocalApiKeyManagerTestSetup` or `LocalApiKeyManagerBaseTest`